tests/ci/commit_status_helper.py

```diff
@@ -255,7 +295,14 @@ def create_ci_report(pr_info: PRInfo, statuses: CommitStatuses) -> str:
         log_urls = None
         if status.target_url is not None:
             log_urls = [status.target_url]
-        test_results.append(TestResult(status.context, status.state, log_urls=log_urls))
+        raw_logs = None
+        if status.description:
+            raw_logs = status.description
+        test_results.append(
+            TestResult(
+                status.context, status.state, log_urls=log_urls, raw_logs=raw_logs
```

Comment:
Don't work yet (no expandable descriptions here https://s3.amazonaws.com/clickhouse-test-reports/54441/6a51d9fd7d2a373741f79ac7f9100225d79ef8bd/ci_running.html), going to check what's wrong